vars/fcosKola.groovy

```diff
@@ -8,13 +8,18 @@
 //    build:             string   -- cosa build ID to target
 //    platformArgs:      string   -- platform-specific kola args (e.g. '-p aws --aws-ami ...`)
 //    extraArgs:         string   -- additional kola args for `kola run` (e.g. `ext.*`)
+//    rerun:             boolean  -- rerun failed tests once
 def call(params = [:]) {
     def cosaDir = utils.getCosaDir(params)
 
     // this is shared between `kola run` and `kola run-upgrade`
     def platformArgs = params.get('platformArgs', "");
     def buildID = params.get('build', "latest");
     def arch = params.get('arch', "");
+    def rerun = ""
+    if (params.get('rerun', false)) {
```

Comment:
Hmm, don't we want to default to rerunning for CI though? So then this would default to `true`, though maybe it's clearer to make the option `disableRerun` instead (and default to `false`).

Reply:
I was going to make another PR to add `rerun: true` to various calls, but I'm fine with changing this to `disableRerun` as well.